data/glossary-tooltip.json

```diff
@@ -1,13 +1,15 @@
 {
   "absolute value": "The distance between a number and \\( 0 \\) on the number line.",
-  "Addition Property of Equality": "When adding the same quantity to both sides of an equation, the equation remains true.",
+  "addition property of equality": "When adding the same quantity to both sides of an equation, the equation remains true.",
   "additive identity": "The number 0, because adding 0 to any number does not change its value.",
   "additive inverse": "The opposite of a number. ",
   "annuity": "An investment that is a sequence of equal periodic deposits.",
+  "area": "Area is a measurement of the total space contained by a shape on a 2-dimensional surface.",
   "arithmetic sequence": "A sequence where the difference between consecutive terms is constant.",
   "association": "In statistics&nbsp;if the value of one of the variables can be used to estimate the value of the\n other.",
   "asymptote": "A line which a graph of a function approaches closely but never touches.",
   "average rate of change": "The change in the outputs divided by the change in the inputs: \\( \\frac{f(b)-f(a)}{b-a} \\).",
+  "axis of symmetry ": "A line that divides a figure, shape, or object about which that figure, shape, or object is symmetrical.",
```

Comment:
I think we need to remove the trailing space on this. Probably it's in the source CSV, but I just updated the code snippet in [the steps](https://docs.google.com/document/d/1z06dCSPw9VwTyFuo2aQUbi7mWp_yA3F9n-bT6PPvE4k/edit#bookmark=id.46ewbu6o4j2z) to use `.strip()` so we just fix it during the conversion. Can you try regenerating with that real quick?